cli/azd/cmd/pipeline.go

```diff
@@ -115,18 +117,20 @@ func newPipelineConfigAction(
 	console input.Console,
 	flags *pipelineConfigFlags,
 	commandRunner exec.CommandRunner,
+	projectManager project.ProjectManager,
 ) actions.Action {
 	pca := &pipelineConfigAction{
 		flags:      flags,
 		azCli:      azCli,
 		credential: credential,
 		manager: pipeline.NewPipelineManager(
-			azCli, azdCtx, env, flags.global, commandRunner, console, flags.PipelineManagerArgs,
+			azCli, azdCtx, env, flags.global, commandRunner, console, projectManager, flags.PipelineManagerArgs,
 		),
-		azdCtx:        azdCtx,
-		env:           env,
-		console:       console,
-		commandRunner: commandRunner,
+		azdCtx:         azdCtx,
+		env:            env,
+		console:        console,
+		commandRunner:  commandRunner,
+		projectManager: projectManager,
```

Comment:
do we need to add this for the `pipelineConfigAction`?

Reply:
Yes it would be required since we require a method on the `ProjectManager`.